services/app-provider/pkg/config/defaults/defaultconfig.go

```diff
@@ -33,7 +33,10 @@ func DefaultConfig() *config.Config {
 		},
 		Driver: "",
 		Drivers: config.Drivers{
-			WOPI: config.WOPIDriver{},
+			WOPI: config.WOPIDriver{
+				WopiFolderURLBaseURL:      "https://localhost:9200/",
```

Comment:
Should we leave that empty? I want the `OCIS_URL` to be used if set. @kobergj

Reply:
No you can leave it in. You should be able to overwrite it by setting `OCIS_URL` or `APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL`